lang/cs/Org.Apache.REEF.Network/StreamingCodec/StreamWithPosition.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Org.Apache.REEF.Network.StreamingCodec
+{
+    public class StreamWithPosition
```

Comment:
I was thinking of the following scenario: What if the second message came while first one was still begin decoded...but I agree there is an issue here wrt. consistency.....Can we make two versions of same stream here...one readable one writable.